sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -265,6 +268,19 @@ builder.Services.AddOpenTelemetry().UseAzureMonitor(o =>
 });
 ```
 
+#### Drop a Metrics Instrument
+
+The Azure Monitor Distro enables metric collection and collects more metrics than specified in the OpenTelemetry specification. 
+If you want to exclude specific instruments from being collected in your application's telemetry use the following code snippet:
+
+```C#
+builder.Services.ConfigureOpenTelemetryMeterProvider(b =>
+    b.AddView(instrumentName: "http.server.active_requests", MetricStreamConfiguration.Drop)
```

Comment:
What is our recommended naming convention for these bound variables? (e.g. "tracing" is used in https://github.com/open-telemetry/opentelemetry-dotnet/blob/d5795c005696e069e61e61f8c3221af8415d3779/docs/trace/getting-started-aspnetcore/Program.cs#L14)

Reply:
Good call out, it should be 
builder.Services.ConfigureOpenTelemetryMeterProvider(metrics =>
    metrics.AddView(instrumentName: "http.server.active_requests", MetricStreamConfiguration.Drop)

Missed the clarity provided here - https://github.com/open-telemetry/opentelemetry-dotnet/pull/4411.

As a follow up PR entire readme needs an update. We might have inconsistent bound variables in several places.

Reply:
Thanks, also check the docs on docs.microsoft.com.

Reply:
I opened an issue to get this cleaned up, both Readme and public docs.
https://github.com/Azure/azure-sdk-for-net/issues/42634